lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -460,6 +460,22 @@ def test_success_v1(self):
         response = self.api_response(api_version=API_V1)
         assert response.data['last_visited_block_id'] == str(self.unit.location)
 
+    # By inheriting from ModuleStoreTestCase and APITestCase this class is atomic by default
+    # By overriding following methods we are making this class non atomic since CourseStatusGET get is non atomic.
+    @classmethod
+    def setUpClass(cls):
+        super(TestCase, cls).setUpClass()  # pylint: disable=bad-super-call
```

Comment:
Oh boy, this feels wrong. Just to clarify, you're intentionally breaking the python method resolution order? There has to be a less brittle way of doing that. What if our super classes also do important stuff besides making us atomic?